variables.py

```diff
@@ -17,10 +17,13 @@
 # -------------------------------------------- 
 
 print("\n------------------- Challenge 1 -------------------\n")
+name = 'JC' 
+grade = 10 
+funFact = 'I like to play video games' 
+print(f"Hi! My name is {name}", type(str))
```

Comment:
The way the `type()` function works is you can put in a variable in the parenthesis and it will print its type:

```suggestion
print(f"Hi! My name is {name}", type(name))
```